static/sass/_snapcraft_custom.scss

```diff
@@ -0,0 +1,12 @@
+@mixin snapcraft-custom {
+  .snapcraft-hero {
+    background-image: url('#{$assets-path}19da7e00-snapcraft-hero-background.png');
```

Comment:
Please use the strip-image pattern: https://docs.vanillaframework.io/en/patterns/strip#strip-image